src/applications/claims-status/components/NeedHelp.jsx

```diff
@@ -26,28 +25,18 @@ export function NeedHelp({ item }) {
   return (
     <va-need-help class="vads-u-margin-top--9">
       <div slot="content">
-        <Toggler toggleName={Toggler.TOGGLE_NAMES.cstFriendlyEvidenceRequests}>
-          <Toggler.Enabled>
+        <p>
+          Call us at <va-telephone contact="8008271000" />. We're here Monday
+          through Friday, 8:00 a.m to 9:00 p.m ET. If you have hearing loss,{' '}
+          <va-telephone contact="711" tty="true" />.
+        </p>
+        {!!item &&
```

Comment:
You probably only need `!!alias` here, I think `!!item` is implicit in `alias`'s existence